enterprise/frontend/src/metabase-enterprise/downloads/downloads_plugin.ts

```diff
@@ -0,0 +1,4 @@
+import { PLUGIN_DOWNLOADS_ENABLED } from "metabase/plugins";
+
+// TODO: implement real logic
+PLUGIN_DOWNLOADS_ENABLED.areDownloadsEnabled = () => true;
```

Comment:
In the next task/pr I'll glue this with the hash parameters coming from  the URL, so that we have a central place with the logic for both dashboards and questions

Reply:
This actually also enables/disables the pdf download button, they're the same flag in the url.

Re `PLUGIN_FEATURE_LEVEL_PERMISSIONS`, that checks for the permissions and it's behind a different feature flag (`advanced_permissions`)

Reply:
Yes it's both for the pdf export button + dashcard results downloads in public/static dashboards/questions.
Maybe `PLUGIN_RESOURCE_DOWNLOADS` would do if I added a comment on the `areDownloadsEnabled` functions?

Reply:
yeah that'd be great.

Reply:
Done in 177927c1329219dc2d4bc3de20c28d0a9cd3ed39 and 23f3491fdee8c1f5aaa0ae0d12fbfee347e5f208